provisioning/image_cache.go

```diff
@@ -1,13 +1,33 @@
 package provisioning
 
 import (
+	"fmt"
+	"net"
+	"net/url"
+	"path"
+	"regexp"
+
+	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	"github.com/openshift/library-go/pkg/operator/resource/resourceapply"
+	"github.com/openshift/library-go/pkg/operator/resource/resourcemerge"
+
+	metal3iov1alpha1 "github.com/openshift/cluster-baremetal-operator/api/v1alpha1"
 )
 
 const (
 	imageCacheSharedVolume = "metal3-shared-image-cache"
+	imageCacheService      = "metal3-image-cache"
+	imageCachePort         = "6181"
```

Comment:
Is this a host port? Should it be registered in https://github.com/openshift/enhancements/blob/master/enhancements/network/host-port-registry.md?

Reply:
Yes; Probably; TIL; and why is 6180 not already in that list?

Reply:
openshift/enhancements#556